extensions/Worker.Extensions.CosmosDB/src/Config/CosmosDBOptions.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.Cosmos;
+
+namespace Microsoft.Azure.Functions.Worker
+{
+    public class CosmosDBOptions
```

Comment:
Can we name this to make it clear it is for the extension? I like `CosmosDBExtensionOptions`